### PR TITLE
Update spaces-overview.md

### PR DESCRIPTION
**Pull request by Michellehbn**
Added Clone Repo button section

Screenshots I added:
Spaces Clone Repo

HTTPS Clone

SSH Clone

### CODE REVIEW

File: docs/hub/spaces-overview.md [hunk 1]
```diff
@@ -84,3 +83,15 @@ In some cases, you might be interested in having programmatic access to the Spac
 * `SPACE_REPO_NAME`: i-like-flan
 * `SPACE_TITLE`: I Like Flan (specified in the README file)
 * `SPACE_ID`: `osanseviero/i-like-flan`
+
+## Clone the Repository
+
+You can easily clone the repo in your Space. Start cloning by clicking on the bullet list dropdown in the top right of your Space page. Select "Clone repository" and then you'll be able to follow the instructions to clone the Space repo to your local machine using HTTPS or SSH.
```
Comment (osanseviero) [inline docs/hub/spaces-overview.md:89]:
```suggestion
You can easily clone the repo in your Space. Start cloning by clicking on the dropdown menu in the top right of your Space page. Select "Clone repository", and then you'll be able to follow the instructions to clone the Space repo to your local machine using HTTPS or SSH. 
```

File: docs/hub/spaces-overview.md [hunk 2]
```diff
@@ -84,3 +83,15 @@ In some cases, you might be interested in having programmatic access to the Spac
 * `SPACE_REPO_NAME`: i-like-flan
 * `SPACE_TITLE`: I Like Flan (specified in the README file)
 * `SPACE_ID`: `osanseviero/i-like-flan`
+
+## Clone the Repository
+
+You can easily clone the repo in your Space. Start cloning by clicking on the bullet list dropdown in the top right of your Space page. Select "Clone repository" and then you'll be able to follow the instructions to clone the Space repo to your local machine using HTTPS or SSH. 
+
+![Spaces Clone Repo](https://github.com/huggingface/hub-docs/assets/83347449/dba41adb-134f-48ef-8de3-36a811c9c883)
```
Comment (osanseviero) [inline docs/hub/spaces-overview.md:91]: These images need to be in https://huggingface.co/datasets/huggingface/documentation-images/tree/main/hub and should be both in dark mode and light mode. See https://github.com/huggingface/hub-docs/blob/main/docs/hub/doi.md?plain=1#L32C1-L35 as an example

**Review by osanseviero — COMMENTED** [review]
Thank you!

File: docs/hub/spaces-overview.md (hunk 2 above)
Comment (Michellehbn) [reply docs/hub/spaces-overview.md]: Added!

**Comment by Michellehbn** [comment]
Images have been updated! Let me know if it's good to go 🙌

**Review by osanseviero — APPROVED** [review]
Looks great! Thanks a lot!

**Pull request merged**